src/service/command/guild-info.ts

```diff
@@ -0,0 +1,213 @@
+ * https://discord.com/developers/docs/resources/guild#guild-object-mfa-level
+ */
+export type GuildMfaLevel = '2FAを要求しない' | '2FAを要求する';
+
+/**
+ * そのギルドのNSFWレベルを定義します。
+ * https://discord.com/developers/docs/resources/guild#guild-object-guild-nsfw-level
+ */
+export type GuildNsfwLevel =
+  | 'デフォルト'
+  | 'iosユーザーに対する制限あり'
+  | '安全'
+  | '年齢制限';
+
+/**
+ * そのギルドのサーバーブーストのティア状況を定義します。
+ * https://discord.com/developers/docs/resources/guild#guild-object-premium-tier
+ */
+export type GuildPremiumTier = 'ティア0' | 'ティア1' | 'ティア2' | 'ティア3';
+
+/**
+ * そのギルドの認証レベルを定義します。
+ * この認証レベルはギルド全体に作用します。
+ * 設定: サーバー設定 → 管理 | 安心設定 → 認証レベル
+ * https://discord.com/developers/docs/resources/guild#guild-object-verification-level
+ */
+export type GuildVerificationLevel =
+  | '制限なし'
+  | '低(メール認証要求)'
+  | '中(作成から5分経過したアカウントのみ)'
+  | '高(限界開発鯖に参加して10分以上経過したアカウントのみ)'
+  | '最高(電話番号認証要求)';
+
+export interface GuildStatsRepository {
+  fetchGuildStats(): Promise<GuildStats | null>;
+}
+
+export class GuildInfo implements CommandResponder {
+  help: Readonly<HelpInfo> = {
+    title: 'ギルド秘書艦',
+    description: '限界開発鯖の情報を持ってくるよ',
+    commandName: ['guildinfo', 'serverinfo'],
+    argsFormat: []
+  };
+
+  constructor(private readonly repo: GuildStatsRepository) {}
+
+  async on(event: MessageEvent, message: CommandMessage): Promise<void> {
+    if (
+      event !== 'CREATE' ||
+      !this.help.commandName.includes(message.args[0])
+    ) {
+      return;
+    }
+
+    const stats = await this.repo.fetchGuildStats();
+    if (!stats) {
+      await message.reply({
+        title: '取得エラー',
+        description:
+          '限界開発鯖の情報を見つけることが出来なかった.... ごめんなさい、力になれなくて'
+      });
+      return;
+    }
+
+    await message.reply(this.buildEmbed(stats));
+  }
+
+  private buildEmbed({
+    afkChannelId,
+    afkTimeout,
+    channelCount,
+    createdAt,
+    emojiCount,
+    id,
+    large,
+    membersCount,
+    mfaLevel,
+    name,
+    nsfwLevel,
+    ownerId,
+    boostTier,
+    roleCount,
+    stickerCount,
+    verificationLevel
+  }: GuildStats) {
+    const fields = [
+      {
+        name: 'サーバー名',
+        value: `${name}\n(${id})`,
+        inline: true
+      },
+      {
+        name: 'オーナー',
+        value: `<@${ownerId}>\n(${ownerId})`,
+        inline: true
+      },
+      {
+        name: 'AFK設定',
+        value: makeAfkChannelMention(afkChannelId, afkTimeout),
+        inline: true
+      },
+      {
+        name: 'チャンネル数',
+        value: `${channelCount}`,
+        inline: true
+      },
+      {
+        name: '絵文字数',
+        value: `${emojiCount}`,
+        inline: true
+      },
+      {
+        name: 'ロール数',
+        value: `${roleCount}`,
+        inline: true
+      },
+      {
+        name: 'ステッカー数',
+        value: `${stickerCount}`,
+        inline: true
+      },
+      {
+        name: '全メンバー数',
+        value: `${membersCount}`,
+        inline: true
+      },
+      {
+        name: '大規模か',
+        value: large ? 'True' : 'False',
```

Comment:
```suggestion
        name: '規模',
        value: large ? '大規模' : '小規模',
```
`True/False` ではなくもう少し具体的に読みやすくにしてほしい